x-pack/plugins/lens/public/visualizations/gauge/toolbar_component/index.tsx

```diff
@@ -87,12 +88,34 @@ export const GaugeToolbar = memo((props: VisualizationToolbarProps<GaugeVisualiz
                 handleChange={(value) => {
                   handleInputChange({
                     ...inputValue,
-                    labelMinor: value.label,
+                    labelMinor: value.mode === 'none' ? '' : value.label,
                   });
                   setSubtitleMode(value.mode);
                 }}
               />
             </EuiFormRow>
+            <EuiFormRow
```

Comment:
do we really want this? I mean, it's another chart switch and we're just about to merge the main one with a layer one. I am ok with it, just asking 🙈 CC @MichaelMarcialis @gvnmagni 
<img width="1345" alt="Screenshot 2024-03-13 at 11 51 40" src="https://github.com/elastic/kibana/assets/4283304/a8f6f566-e130-4dbb-9014-23178ee11b37">

Reply:
Good comment, I think we can hide/remove it for now, and we can get back to it once we defined the new chart switcher design https://github.com/elastic/kibana/pull/177766#discussion_r1523555696

Reply:
I agree as well. This should probably be removed until a more holistic approach to visualization type subcategories can be addressed, as discussed in the comments above.

Reply:
Removed type option in 4e1379af0b1d40223a1700fb2bcf3c8eb80b5fb3